{
  "id": "CVE-2017-6299",
  "assigner": "cve@mitre.org",
  "Published": "2017-02-24T03:59:00.297Z",
  "Modified": "2019-10-02T22:03:26.223Z",
  "last-modified": "2019-10-02T22:03:26.223Z",
  "summary": "An issue was discovered in ytnef before 1.9.1. This is related to a patch described as \"2 of 9. Infinite Loop / DoS in the TNEFFillMapi function in lib/ytnef.c.\"",
  "access": {
    "authentication": "NONE",
    "complexity": "MEDIUM",
    "vector": "NETWORK"
  },
  "impact": {
    "availability": "PARTIAL",
    "confidentiality": "NONE",
    "integrity": "NONE"
  },
  "cvss": 4.3,
  "exploitabilityScore": 8.6,
  "impactScore": 2.9,
  "cvss-vector": "AV:N/AC:M/Au:N/C:N/I:N/A:P",
  "references": [
    "http://www.debian.org/security/2017/dsa-3846",
    "http://www.openwall.com/lists/oss-security/2017/02/15/4",
    "http://www.securityfocus.com/bid/96423",
    "https://github.com/Yeraze/ytnef/pull/27",
    "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/LFJWMUEUC4ILH2HEOCYVVLQT654ZMCGQ/",
    "https://www.x41-dsec.de/lab/advisories/x41-2017-002-ytnef/"
  ],
  "vulnerable_configuration": [
    "cpe:2.3:a:ytnef_project:ytnef:1.7:*:*:*:*:*:*:*",
    "cpe:2.3:a:ytnef_project:ytnef:1.8:*:*:*:*:*:*:*",
    "cpe:2.3:a:ytnef_project:ytnef:1.9:*:*:*:*:*:*:*"
  ],
  "vulnerable_product": [
    "cpe:2.3:a:ytnef_project:ytnef:1.7:*:*:*:*:*:*:*",
    "cpe:2.3:a:ytnef_project:ytnef:1.8:*:*:*:*:*:*:*",
    "cpe:2.3:a:ytnef_project:ytnef:1.9:*:*:*:*:*:*:*"
  ],
  "vendors": [
    "ytnef_project"
  ],
  "products": [
    "ytnef"
  ],
  "vulnerable_product_stems": [
    "cpe:2.3:a:ytnef_project:ytnef"
  ],
  "vulnerable_configuration_stems": [
    "cpe:2.3:a:ytnef_project:ytnef"
  ],
  "cwe": "CWE-835",
  "vulnerable_configuration_cpe_2_2": []
}